.azure/scripts/coverage.yml

```diff
@@ -11,7 +11,7 @@ steps:
 - script: |
     python setup.py test_java
     pip install gcovr pytest-cov -r test-requirements.txt
-    pip install numpy
+    pip install numpy setuptools
```

Comment:
Needed here because we use `setup.py devlop` directly, and therefore there is no automatic build dep installation.